recipe/meta.yaml

```diff
@@ -33,6 +36,6 @@ about:
   home: https://aiopg.readthedocs.io
   license: BSD-2-Clause
   license_family: BSD
-  license_file: LICENSE.txt
+  license_file: LICENSE
   summary: "Postgres integration with asyncio."
   dev_url: https://github.com/aio-libs/aiopg
```

Comment:
We need to add a `doc_url` to the recipe

https://github.com/AnacondaRecipes/aiopg-feedstock/blob/7fc0166a16ab43c920e620a9c95251703efadb42/recipe/meta.yaml#L41

Reply:
added doc url